src/components/tables/DataTable.vue

```diff
@@ -51,7 +51,7 @@ export default {
   },
   methods: {
     initTable: function() {
-      $.fn.dataTable.moment('D MMM YYYY');
+      $.fn.dataTable.moment(moment().creationData().locale._longDateFormat.LL);
```

Comment:
Let's use `ll` so we're showing `Aug` instead of `August`.

```suggestion
      $.fn.dataTable.moment(moment().creationData().locale._longDateFormat.ll);
```